app/src/main/java/org/fossasia/susi/ai/rest/ClientBuilder.java

```diff
@@ -36,10 +36,6 @@ public ClientBuilder() {
         createSusiService();
     }
 
-    private static void init() {
-        susiService = createApi(SusiService.class);
-    }
-
     private static <T> T createApi(Class<T> clazz) {
```

Comment:
How many places is this function used?

Reply:
@iamareebjamal As requested, I moved that to the getSusiApi ( ) and it's not being used anywhere else.

Reply:
So remove this

Reply:
i already did, it shows red mark of removal

Reply:
guide me if I can't catch what you are saying..thanks

Reply:
@iamareebjamal this is used at only one place below in the getSusiApi() function

Reply:
Where is it used?

Reply:
Yeah, so move it there

Reply:
Not moved, please clear the topic up before committing. Saves the time of maintainers and collaborators

Reply:
@codedsun If you moved it already, this comment would have automatically collapsed, so it is not moved. Look at what @batbrain7 has mentioned from **2 days ago** about where it's used

Reply:
@iamareebjamal  You said to move the init() function to getSusiApi() function, this is what I did and here you have commented on the ***Removed LINES*** code which shows clear red bar on the lines, and @batbrain7 just commented the usage of this function. I still couldn't get what is to be removed as the code branch doesn't contain this what you have commented. I have these methods now as in PR
How can the comments be gone, when the code is already removed earlier? 
```
package org.fossasia.susi.ai.rest;

...
public class ClientBuilder {

    private static Retrofit retrofit;
    private static SusiService susiService;

    /**
     * Instantiates a new Client builder.
     */
    public ClientBuilder() {
        createSusiService();
    }

    private static <T> T createApi(Class<T> clazz) {
        return retrofit.create(clazz);
    }

    /**
     * Create susi service.
     */
    public static void createSusiService() {

        try {
            retrofit = RetrofitInstance.getRetrofit();
            susiService = getSusiApi();
        } catch (IllegalArgumentException e) {
            e.printStackTrace();
        }
    }

    /**
     * Gets susi api.
     *
     * @return the susi api
     */
    public static SusiService getSusiApi() {
        if (susiService == null) {
            if (retrofit == null)
                retrofit = RetrofitInstance.getRetrofit();
            susiService = createApi(SusiService.class);
        }
        return susiService;
    }

    ...
}
```

Reply:
If I am wrong please guide me! Thanks

Reply:
Please see the line where I have added this comment. Please see the name of the method about which I am talking about. Please look at the comments below from @batbrain7 

@batbrain7 Please help. Please tell @codedsun about which method I am talking about here, and what you have commented down below

Reply:
If you don't know, @codedsun, github collapses the comment which has been resolved, so once you fulfill the change I have requested, it will be collapsed automatically. You'll see it for yourself

Reply:
Also, I have absolutely, **not** commented on removed lines which you are so confused about, I have commented on a function which is already there in this commit and asked you to remove it. This is also what @batbrain7 commented